doc/chapter-firststeps.asciidoc

```diff
@@ -1,10 +1,10 @@
-After you've installed newsboat, you can run it for the first time by typing
+After you've installed Newsboat, you can run it for the first time by typing
 `newsboat` on your command prompt. This will bring you the following message:
 
 ----
 Error: no URLs configured. Please fill the file /home/ak/.newsboat/urls with RSS feed URLs or import an OPML file.
 
-newsboat 2.10
+Newsboat 2.10
```

Comment:
This is a listing of the program's output, and we'll have to change some code to change the listing. For now, please revert this change.